app/views/shared/_topic_page_header.html.erb

```diff
@@ -0,0 +1,20 @@
+<%
+   two_thirds ||= false
+   margin_bottom ||= 0
+
+   header_wrapper_classes = %w(topic-page__header-wrapper)
+   header_wrapper_classes << "govuk-!-margin-bottom-#{margin_bottom}" if !margin_bottom.zero?
+
+   column_classes = %w()
+   column_classes << two_thirds ? "govuk-grid-column-two-thirds" : "govuk-grid-column-full"
+%>
+
+<div class="<%= header_wrapper_classes.join(' ') %>">
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="<%= column_classes.join(' ') %>">
```

Comment:
This outputs `false` instead of the class name.

Reply:
was missing some parenthesis, should be working now!